src/ServiceContainer/SymfonyExtension.php

```diff
@@ -182,6 +188,17 @@ private function loadBootstrap(?string $bootstrap): void
         require_once $bootstrap;
     }
 
+    private function loadEnv(array $config)
+    {
+        $env = $config['environment'] ?? $_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? 'test';
+
+        try {
+            (new Dotenv())->bootEnv(basename(dirname(__DIR__)) . '/../.env', $env);
```

Comment:
All the other IO-related functions in this file use CWD as the base - I wouldn't switch it to `__DIR__` just in this PR as it introduces a non-standard behaviour. Could we go with the following?

```suggestion
            (new Dotenv())->bootEnv('.env', $env);
```